change/@ni-nimble-components-7612f990-8c7c-49a4-bfa2-53d913115c1d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like we regressed in the actions performance. In the table story with a large data set open and action menu and notice the slow performance. There was a workaround added to avoid that perf issue and it looks like it regressed:
![image](https://user-images.githubusercontent.com/1588923/229844377-b59e15a9-687e-4872-af8e-9b5232a845c0.png)

Reply:
https://github.com/ni/nimble/blob/2d95346d6afab9d204d5796acd07825d7cec8cbe/packages/nimble-components/.storybook/preview.js#L25-L30

Reply:
Additional context: https://github.com/ni/nimble/pull/1056#discussion_r1110159708

Reply:
Great catch! It appears the meaning of the `depth` parameter has changed in Storybook 7 and now I have to set `depth: 1` to achieve the same result that `depth: 6` previously did. (Most likely the structure of their internal event data changed and they're doing something to hide that from us but didn't update the depth logic).

I changed the depth to 1 for now and filed https://github.com/storybookjs/storybook/issues/21933 to find out if this was a regression. I'll add it to the list of items to follow up on in #1146.

Reply:
Excellent bug report 👍